libs/utils/gtest/CMakeLists.txt

```diff
@@ -15,21 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set(CMAKE_CXX_STANDARD 17)
```

Comment:
Does this mean that `test_utils` is now C++14? If so, `utils/CMakeLists.txt` should also be updated:

```CMake
        if (CELIX_CXX17) #utils tests are C++17
            add_subdirectory(gtest)
        endif()

```